snmp/datadog_checks/snmp/config.py

```diff
@@ -183,6 +186,9 @@ def refresh_with_profile(self, profile):
         metric_tags = profile['definition'].get('metric_tags', [])
         tag_oids, parsed_metric_tags = self.parse_metric_tags(metric_tags)
 
+        device = profile['definition'].get('device', {})
```

Comment:
I think it might be good to validate that there is no invalid attributes (attributes other than `vendor`).

Reply:
Attributes that are not in `SUPPORTED_DEVICE_TAGS` will be ignored. For now, only `vendor` is supported.